rotkehlchen/api/v1/resources.py

```diff
@@ -2953,3 +2954,13 @@ class DefiMetadataResource(BaseMethodView):
 
     def get(self) -> Response:
         return self.rest_api.get_defi_metadata()
+
+
+class ExportHistoryEventResource(BaseMethodView):
+
+    post_schema = ExportHistoryEventSchema()
+
+    @require_loggedin_user()
+    @use_kwargs(post_schema, location='json_and_query')
+    def post(self, filter_query: 'HistoryBaseEntryFilterQuery', directory_path: Path) -> Response:
+        return self.rest_api.export_history_events(filter_query=filter_query, directory_path=directory_path)  # noqa: E501
```

Comment:
about the path for exporting the file I'm not sure what was the discussion... can the frontend provide it for docker and electron? in some endpoints we need to methods, one to download from the browser and one to save in the fs. Have you coordinated on this?

Reply:
seen, I can export, I want to add a query param that will be used for this, something like ?download=true, which would trigger a download instead of an export alone

Reply:
but have you asked the frontend about this? If they can handle it or if they have any special requirement. Can you contact them in discord to coordinate in this topic?